docs/content/docs/recipes/angular.md

```diff
@@ -1,6 +1,285 @@
 ---
 title: Using Fluid with Angular
 menuPosition: 2
-status: unwritten
-discussion: 5464
+aliases:
+  - "/start/angular-tutorial/"
 ---
+
+In this tutorial, you'll learn about using the Fluid Framework by building a simple application that enables every client of the application to change a dynamic time stamp on itself and all other clients almost instantly. You'll also learn how to connect the Fluid data layer with a view layer made in [Angular](https://angular.io).
+
+To jump ahead into the finished demo, check out the [Create React App demo in our FluidExamples repo](https://github.com/microsoft/FluidExamples/tree/main/angular-demo).
+
+The following image shows the time stamp application open in four browsers. Each has a button labeled **click** and beside it a Unix epoch time. The same time is in all four. The cursor is on the button in one browser.
+
+![Four browsers with the Timestamp app open in them.](https://fluidframework.blob.core.windows.net/static/images/Four-clients-1.PNG)
+
+The following image shows the same four clients one second after the **click** button was pressed. Note that the timestamp has updated to the very same time in all four browsers.
+
+![Four browsers with the Timestamp app open in them one second after the button has been pushed.](https://fluidframework.blob.core.windows.net/static/images/Four-clients-2.PNG)
+
+{{< callout note >}}
+
+This tutorial assumes that you are familiar with the [Fluid Framework Overview]({{< relref "/docs/_index.md" >}}) and that you have completed the [QuickStart]({{< relref "quick-start.md" >}}). You should also be familiar with the basics of [Angular](https://angular.io), [creating Angular projects](https://angular.io/guide/setup-local), and [Angular Hooks](https://angular.io/guide/lifecycle-hooks).
+
+{{< /callout >}}
+
+
+## Prerequisites
+
+1. Node.js must be installed on your local machine. To install, follow the instructions [here](https://nodejs.org/en/download/).
+1. Angular CLI must be installed on your local machine. To install, run the following command (after Node.js is installed).
+
+    ```dotnetcli
+    npm install -g @angular/cli
+    ```
+
+## Create the project
+
+1. Open a Command Prompt and navigate to the parent folder where you want to create the project; e.g., `c:\My Fluid Projects`.
+1. Run the following command at the prompt.
+    ```dotnetcli
+    ng new fluid-angular-tutorial
+    ```
+
+1. The project is created in a subfolder named `fluid-angular-tutorial`. Navigate to it with the command `cd fluid-angular-tutorial`.
+1. The project uses two Fluid libraries:
+
+    |Library |Description |
+    |---|---|
+    |fluid&#x2011;framework&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;    |Contains the SharedMap [distributed data structure]({{< relref "dds.md" >}}) that synchronizes data across clients. *This object will hold the most recent timestamp update made by any client.*|
```

Comment:
@tylerbutler These strings of non-breaking spaces is so weird, and they show up on the docs site.  Is there a way to set a right column margin for this formatting?

Reply:
I think we should just remove them.

Reply:
I found the root cause of the table layout problem, and I'll fix in a different PR. You can remove all the non-breaking spaces.